src/models/BaseLoginModel.js

```diff
@@ -12,9 +12,10 @@
 
 define([
   'okta',
-  'vendor/lib/q'
+  'vendor/lib/q',
+  'util/Enums',
 ],
-function (Okta, Q) {
+function (Okta, Q, Enums) {
 
   var _ = Okta._;
   var KNOWN_ERRORS = [
```

Comment:
Interesting why `AuthPollStopError` was added here since we will return before that. Curious why it was added here. Should we add `AUTH_STOP_POLL_INITIATION_ERROR` in case this array is going to be used elsewhere, since this is also going to be a known error?

Reply:
Lol yeah this has to be cleaned up.  I checked its usage and seems like it doesnt make sense to add it to Known errors array.